q2_composition/_ancombc.py

```diff
@@ -72,10 +72,27 @@ def _leaf_collector(term):
     return _leaf_collector(term[1]) + _leaf_collector(term[2])
 
 
+def _get_ref_level_defaults_from_formula_terms(metadata, term,
+                                               reference_levels):
+    term_alpha_value = (metadata.get_column(term)
+                        .to_dataframe()
+                        .sort_values(term)[term][0])
+    ref_level_pair = term + '::' + str(term_alpha_value)
+    reference_levels.append(ref_level_pair)
+
+    return reference_levels
+
+
```

Comment:
Since `reference_levels` is a list, and lists are mutable, the value `reference_levels` has after appending on line 81 will also be its value outside of this function. Lists are basically pass-by-reference, not pass-by-value, so mutating the state of a list inside of a function like this and then returning the list is redundant, but it might be worth keeping for clarity.

Reply:
That makes sense, thanks @Oddant1 - I think I'll leave it as-is for clarity's sake, unless @ebolyen thinks it's unnecessary?